cdap-data-fabric/src/test/java/co/cask/cdap/data2/metadata/store/MetadataStoreTest.java

```diff
@@ -188,6 +190,35 @@ public void testPublishingDisabled() throws InterruptedException {
     cConf.setBoolean(Constants.Metadata.UPDATES_PUBLISH_ENABLED, publishEnabled);
   }
 
+  @Test
+  public void testSearchWeight() throws Exception {
+    Id.Program flow1 = Id.Program.from("ns1", "app1", ProgramType.FLOW, "flow1");
+    Id.Stream stream1 = Id.Stream.from("ns1", "s1");
```

Comment:
`MetadataDatasetTest.testSearchOnTags()` test already verifies that the number of tags returned is according to the tag matches. In `DefaultMetadataStore`, we only count the number of `MetadataEntry`s returned by `MetadataDataset.search()`. Hence testing for tags is strictly not required here.